### DISCUSSION TITLE
Can't sync with Miband 6

### DISCUSSION THREAD

**Discussion by dagord** [NightscoutFoundation/xDrip#2220] [Q&A]
Hello

I recently switched from Miband 4 to Miband 6, and since then I'm not able to get data from XDrip+.

I'm on the same setup as before, which means that I'm using the **Notify for Mi Band** app (instead of the official **Mi Fit**) with the auth code I got using the methods suggested on freemyband.com.

In **XDrip+**, form the "MiBand" settings sections, I replaced the old mac address with the new one but there's no field for entering the auth code!
![1657724651984](https://user-images.githubusercontent.com/21258057/178769037-4b7b3fd7-f800-49ec-bc29-9758c6b36757.jpg)

Under system status screen, I see that the MiBand is connected but not authenticated.
![1657724651969](https://user-images.githubusercontent.com/21258057/178769082-29c2bafa-0ba1-40e4-ac3c-a79152bbadde.jpg)

I get a popup message saying "MiBand authentication failed due to authentication timeout", but from the logs I can't find any relevant information.
![1657724651950](https://user-images.githubusercontent.com/21258057/178769105-0c1b1690-c79c-4000-b8e0-1dc62c4275bc.jpg)

Of course I can still get alarm notifications (having properly configured **Notify for Mi band**), but I can't push data natively from XDrip+, so I can't have the watchface updates as I used before, a feature that was very useful.

As I understand, official support is only up to Miband 5, so there's some chance to get the Miband 6 working? Or am I missing something?
Thanks in advance

**Comment by Navid200** [comment]
This is where you should start with Mi Band:  https://bigdigital.home.blog/2022/06/16/watchdrip-a-new-application-for-xdrip-watch-integration/

If you find bugs or if you like to request a new feature, you should submit here:
https://github.com/bigdigital/watchdrip/issues

If it turns out that the issue is with xDrip, the developer of the companion app can then submit a request to xDrip or even open a PR himself.

**Reply by dagord** [reply]
Sounds good! I didn't know about that side project!
Thank you!
